python/dgl/data/qm9_edge.py

```diff
@@ -0,0 +1,404 @@
+"""QM9 dataset for graph property prediction (regression)."""
+"""When building this class, we partially refer to the implementation from https://github.com/Jack-XHP/DGL_QM9EDGE """
+
+import os
+import numpy as np
+
+from .dgl_dataset import DGLDataset
+from .utils import download, extract_archive, _get_dgl_url
+from ..convert import graph as dgl_graph
+from .. import backend as F
+
+try:
+    import rdkit
+    from rdkit import Chem
+    from rdkit.Chem.rdchem import HybridizationType
+    from rdkit.Chem.rdchem import BondType as BT
+    from rdkit import RDLogger
+    RDLogger.DisableLog('rdApp.*')
+except ImportError:
+    rdkit = None
+HAR2EV = 27.2113825435
+KCALMOL2EV = 0.04336414
+conversion = F.tensor([
+    1., 1., HAR2EV, HAR2EV, HAR2EV, 1., HAR2EV, HAR2EV, HAR2EV, HAR2EV, HAR2EV,
+    1., KCALMOL2EV, KCALMOL2EV, KCALMOL2EV, KCALMOL2EV, 1., 1., 1.
+])
+
+
+class QM9EdgeDataset(DGLDataset):
+    r"""QM9Edge dataset for graph property prediction (regression)
+    This dataset consists of 130,831 molecules with 19 regression targets.
+    Node means atom and edge means bond.
+    
+    This dataset differs from QM9Dataset in the following points:
```

Comment:
"QM9Dataset" -> ":class:`~dgl.data.QM9Dataset`" for hyperlink in rendering the documentation